bindings/python/test/trajectory/orbit/models/test_tabulated.py

```
@@ -265,6 +268,70 @@ def test_constructor_orbit_tabulated_sucess(
+                CoordinateSubset.mass(): Interpolator.Type.Linear,
+            },
+        )
+
+        assert tabulated.is_defined()
+
+    def test_constructor_with_interpolation_types_failure(
+        self,
+        test_states: list[State],
+    ):
+        # A coordinate subset present in the states but missing from the map must raise.
+        with pytest.raises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)